.github/workflows/ci.yml

```
@@ -0,0 +1,44 @@
+name: CI
+
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
```

This made em realise we still have `master` branch! Have just removed

```suggestion
    branches: [ main ]
  pull_request:
    branches: [ main ]
```